source/Calamari.Shared/Integration/Certificates/CryptoKeySecurityAccessRules.cs

```diff
@@ -1,18 +1,260 @@
 ﻿using System;
 using System.Collections.Generic;
 using Calamari.Integration.Certificates.WindowsNative;
-#if WINDOWS_CERTIFICATE_STORE_SUPPORT
 using System.Linq;
+using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
+using System.Security;
 using System.Security.AccessControl;
 using System.Security.Cryptography;
 using System.Security.Cryptography.X509Certificates;
+using System.Security.Principal;
 using static Calamari.Integration.Certificates.WindowsNative.WindowsX509Native;
-#endif
+
 
 namespace Calamari.Integration.Certificates
 {
-#if WINDOWS_CERTIFICATE_STORE_SUPPORT
+    #if !NETFX
+       [Flags]
+    public enum CryptoKeyRights
```

Comment:
These classes are all missing from the netcore libraries and so need to be C+P so that we can continue to use through our unmanaged code calls

Reply:
not gonna lie, took me longer than I care to get `C+P` 😆